perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -193,11 +194,15 @@ def FillDevice(vm, disk, fill_size):
   vm.RemoteCommand(command)
 
 
-BENCHMARK_INFO = {'name': 'fio',
-                  'description': 'Runs fio in sequential, random, read '
-                                 'and write modes.',
-                  'scratch_disk': True,
-                  'num_machines': 1}
+BENCHMARK_NAME = 'fio'
+BENCHMARK_CONFIG = """
+fio:
+  description: Runs fio in sequential, random, read and write modes.
+  vm_groups:
+    default:
+      vm_spec: *default_single_core
+      disk_spec: *default_500_gb
```

Comment:
Comparing fio benchmarks on 500 GB disks can be very misleading, because GCP and AWS (for instance) give different IOPS/GB. I can fix this in a later commit, if you'd like.